src/main/java/xyz/aadev/generitech/common/tileentities/power/DistributePowerToFace.java

```diff
@@ -44,18 +44,18 @@ private DistributePowerToFace() {
 
     public static void transferPower(BlockPos pos, World worldIn, long ratetranfer, BaseTeslaContainer container, int[] faces) {
 
-        long inputba = distributePowerToAllFaces(worldIn, pos, ratetranfer, true,faces);
+        long inputba = distributePowerToAllFaces(worldIn, pos, ratetranfer, true, faces);
         long test = inputba / ratetranfer;
         if (test == 0) test = 1;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00122)